src/cascadia/TerminalApp/AppLogic.h

```diff
@@ -98,6 +101,8 @@ namespace winrt::TerminalApp::implementation
         winrt::com_ptr<LanguageProfileNotifier> _languageProfileNotifier;
         wil::unique_folder_change_reader_nothrow _reader;
 
+        TerminalApp::ContentManager _contentManager{ *winrt::make_self<implementation::ContentManager>() };

```

Comment:
fwiw you should be able to use `winrt::make` instead of `*winrt::make_self`; the only difference between `make` and `_self` is that `_self` returns a `com_ptr` and non-`_self` returns the projection type.

Since all you're getting is the projection type... use `make`!

Reply:
Also it is important to note that this happens during construction. NBD since it's free-threaded and non-xaml, but good to remember.